ansible/playbooks/roles/repository/files/download-requirements/src/command/dnf_download.py

```diff
@@ -32,4 +32,19 @@ def download_packages(self, packages: List[str],
         args.append('-y')
         args.extend(packages)
 
-        self.run(args)
+        output_stdout = self.run(args).stdout
+        output_stderr = self.run(args).stderr
+
+        def output_handler(output_stdout: str, output_stderr: str):
```

Comment:
You use it only once, so no need for a function. Just add those 2 ifs from lines: 43-46 to the main body of download_packages(self).